react-intervaler/src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import './App.scss';
```

Comment:
better to use absolute instead of relative paths. Though this is all at root so :shrug: If you want, look into how to set that up in a node/react project.